Makefile

```diff
@@ -195,11 +186,15 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-image
```

Comment:
Is the comment block still up-to-date?

Reply:
Pretty sure it is not even without checking. I am happy to do a follow up cleanup PR, so that the release workflow can be tested with this pr getting merged

Reply:
fine by me :)

Reply:
I have removed the stale comment albeit, we still use semver.